tools/deploy/deployInvoker

```diff
@@ -1,305 +0,0 @@
-#!/usr/bin/env python
-
-#
-# Copyright 2015-2016 IBM Corporation
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-# http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-#
-
-#
-# >>>>> If this script is moved, the path manipulation needs to be adjusted.  See below. <<<<<
-#
-# Usage:
-#   deployInvoker                 deploy all invokers from INVOKER_HOSTS
-#   deployInvoker index host      deploy an invoker with the given index on the given host
-#   deployInvoker --kill ...      kill rather than deploy
-# Returns:
-#  0                     if all were deployed successfully
-#  >0                    a count of failed deploys
-#
-# A pull, teardown, and isAlive check is performed as well.
-# The script is synchronous and will not return until deployment has completed.
-# The list of invoker hosts is obtained from the Whisk property file.
-#
-
```

Comment:
@perryibm do you want to keep this? can we accomplish the same with the playbooks?